brats/losses.py

```diff
@@ -34,3 +35,69 @@ def __call__(self, prediction: torch.Tensor,
             torch.Tensor: Computed Dice Loss
         """
         return torch.sum(1 - F.dice(prediction, target, self.epsilon))
+
+
+class NLLLossOneHot(nn.Module):
```

Comment:
With increasing number of loss classes, I think it would be beneficial to create an interface for it.

Reply:
Created an interface